driver_test.go

```diff
@@ -21,3 +21,12 @@ func TestSQLOpen(t *testing.T) {
 	defer db.Close()
 	a.NoError(err)
 }
+
+func TestQuerySettings(t *testing.T) {
+	a := assert.New(t)
+	db, err := sql.Open("maxcompute", cfg4test.FormatDSN())
+	a.NoError(err)
+	db.Driver().(*MaxcomputeDriver).SetQuerySettings(map[string]string{"odps.sql.mapper.split.size": "16"})
```

Comment:
Does this test verify the hints is effective?

Reply:
Shall we also test `odps.sql.mode = script`?

Reply:
> Does this test verify the hints is effective?

No, it's not easy to test `odps.sql.mapper.split.size ` on the client-side. But maybe test UDF script is good way, will add this one.